proposals/p1964.md

```diff
@@ -0,0 +1,351 @@
+        the difference between the code points.
+
+    In cases where the `+` and `-` operators are used with a character literal
+    and a non-integer non-literal, the character literal should be converted to
+    the type of the other operand if possible. For example, in the expression
+    `w - 'a'`, where `w` is of type `Char8`, the `'a'` literal will be converted
+    to type `Char8`.
+
+There is intentionally no implicit conversion from character literals to integer
+types, but explicit conversions are permitted between character literals
+representing a single code point and integer types. Carbon will separate the
+integer types from character types entirely.
+
+## Rationale
+
+This proposal supports the goal of making Carbon code
+[easy to read, understand, and write](/docs/project/goals.md#code-that-is-easy-to-read-understand-and-write).
+Adding support for a specific character literal supports clean, readable,
+concise use and is a much more familiar concept that will make it easier to
+adopt Carbon coming from other languages. Have a distinct character literal will
+also allow us support useful operations designed to manipulate the literal's
+value. When working with `String`, we use the `+` operator to concatenate
+multiple `String`s, but say we wanted to advance a character to the next
+literal. Using a `String` will produce a type error, as we are misusing the `+`
+operator: `"a" + 1`. However with a character literal, we can support operations
+for these use cases:
+
+```
+var b: u8;
+
+b = 'a' + 1;
+b + 1 == 'c';
+
+```
+
+See [Operations](#operations) and
+[No Distinct Character Literal](#no-distinct-character-literal) for more
+information.
+
+Further, this design follows other standards set in place by previous proposals.
+For example following the
+[String Literals: Escaping Sequence](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0199.md#escape-sequences-1)
+and representing characters as integers with the behaviour inline with
+[Integer Literals](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0143.md).
+
+This also supports our goal for
+[Interoperability with and migration from existing C++ code](/docs/project/goals.md#interoperability-with-and-migration-from-existing-c-code)
+by ensuring that every kind of character literal that exists in C++ can be
+represented in a Carbon character literal. This is done in a way that is natural
+to adopt, understand, easy to read by having explicit character types mapped to
+the C++ character types and the correct associated encoding.
+
+Finally, the choice to use Unicode and UTF-8 by default reflects the Carbon goal
+to prioritize
+[modern OS platforms, hardware architectures, and environments](/docs/project/goals.md#modern-os-platforms-hardware-architectures-and-environments).
+This reflects the
+[growing adoption of UTF-8](https://en.wikipedia.org/wiki/UTF-8#Adoption).
+
+## Alternatives considered
```

Comment:
We should add an alternative of supporting grapheme clusters, and of supporting non-code-point code-units.